### PR TITLE
Add more docs

### PR DESCRIPTION
**Pull request by HeinrichHartmann**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,29 @@ An implementation of Circonus log-linear histograms.
...
+    make docs
+
+This requires doxygen to be available on your system.
+
+## Python bindings
+
+Build and install the C library as described above.
+Then use the following command to install the python library.
+
+    cd src/python
```
Comment (cody271) [inline README.md]: All the shell commands in the README should be consistently quoted as such. For example:
```
$ cd src/python
$ python setup.py install
```

File: README.md [hunk 2]
```diff
@@ -8,3 +8,29 @@ An implementation of Circonus log-linear histograms.
...
+    python setup.py install
+
+## Lua bindings
+
+This repository contains lua bindings for libcircllhist, which are generated my `make`,
```
Comment (cody271) [inline README.md]: Typos:

"which are generated **by** `make`"

"and will be installed **by** `make install`"

**Comment by cody271** [comment]
Also- figured out the Doxygen mainpage stuff:

```diff
diff --git a/src/circllhist.h b/src/circllhist.h
index 304cb5f..6ddb015 100644
--- a/src/circllhist.h
+++ b/src/circllhist.h
@@ -29,6 +29,10 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+/*! \mainpage A C implementation of Circonus log-linear histograms
+ * \ref circllhist.h
+ */
+
 #ifndef CIRCLLHIST_H
 #define CIRCLLHIST_H
 

```

**Comment by HeinrichHartmann** [comment]
The comments in circllhist.c are just internal notes. No API documentation. So this is fine.

File: README.md (hunk 1 above)
Comment (HeinrichHartmann) [reply README.md]: The quoting of this snippet is consistent with the ones found above.
Adding `$` at the beginning, will break copy-paste. So I'd rather not add it in.

Comment (cody271) [reply README.md]: On second look at the README, all good. GitHub actually does the right thing here and interprets the indented lines as shell commands, no quoting markup necessary.

**Comment by cody271** [comment]
All fixed. LGTM.

**Pull request merged**